packages/graphql-server/src/constants/api.ts

```diff
@@ -7,6 +7,7 @@ export const URLS = {
   appointments: '/appointments',
   identityChecks: '/identityChecks',
   negotiators: '/negotiators',
+  propertyTypes: '/configuration/appointmentTypes',
```

Comment:
So I think I have screwed up here with my ticket - really sorry, I thought the name was PropertyTypes but should be AppointmentTypes. Apologies. 


Also, is there an easy way to expand this endpoint so we can capture all of the configuration options by passing in an extra param eg `/configuration/:type` They all should return the same key value pairs so should be possible right? Would be a really nice enhancement if so :-)